core/common/src/main/java/alluxio/grpc/ServiceVersionClientServiceHandler.java

```diff
@@ -51,7 +51,7 @@ public ServiceVersionClientServiceHandler(
   @SuppressFBWarnings(value = "DB_DUPLICATE_SWITCH_CLAUSES")
   public void getServiceVersion(GetServiceVersionPRequest request,
       StreamObserver<GetServiceVersionPResponse> responseObserver) {
-    if (mStandbyRpcEnabled
+    if (!request.getAllowOnStandbyMasters() && mStandbyRpcEnabled
```

Comment:
just a sanity check that default is `getAllowOnStandbyMasters() == false` right? Add a brief comment for that.